.github/workflows/image-arm.yaml

```diff
@@ -103,8 +103,14 @@ jobs:
           TAG: "latest"
           COSIGN_YES: true
         run: |
-          docker push "$IMAGE:$TAG" # Otherwise .RepoDigests will be empty for some reason
-          cosign sign $(docker image inspect --format='{{index .RepoDigests 0}}' "$IMAGE:$TAG")
+          docker push "$IMAGE:$TAG"
+          image_ref=$(docker image inspect --format='{{index .RepoDigests 0}}' "$IMAGE:$TAG")
+          cosign sign $image_ref
```

Comment:
I woudl remove this and `cosign sign ${image_ref}.sbom` and just use `cosign sign --attachment sbom $image_ref` to see if it signs the whole thing, so we only have to sign once